tensorflow/compiler/xla/service/cpu/BUILD

```diff
@@ -149,7 +150,7 @@ cc_library(
 
 cc_library(
     name = "simple_orc_jit",
-    srcs = ["simple_orc_jit.cc"],
+    srcs = ["simple_orc_jit.cc"] + if_windows(["windows_compatibility.h"]),
```

Comment:
Once you guard the whole file under `_MSC_VER` I think you'll also have to remove the `if_windows` bit and unconditionally list `windows_compatibility.h` in `srcs`.

Reply:
Done